tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```diff
@@ -79,6 +79,13 @@ RUN cd /openedx/requirements/ \
 {% for extra_requirement in ECOMMERCE_EXTRA_PIP_REQUIREMENTS %}RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,sharing=shared,uid=${APP_USER_ID} {% endif %}pip install '{{ extra_requirement }}'
 {% endfor %}
 
+{% if ECOMMERCE_ATLAS_PULL %}
+# Pull translations. Support the OEP-58 proposal behind a feature flag until it's fully implemented.
+RUN find ecommerce/conf/locale -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
```

Comment:
This line is discouraged from being used in the _Tutor plugins_. See Regis comment: https://github.com/overhangio/tutor-discovery/pull/41#discussion_r1316002004 .

```suggestion
```

Reply:
done